tasks/main.yml

```diff
@@ -38,6 +38,17 @@
     msg: "Creating partition of size {{ size }} on {{ resolved_disks.results[0]['device'] }}"
   when: (use_partitions and device_type != 'disk') or device_type == "partition"
 
+- name: Generate default name for LVG 
+  lvm_gensym:
+  register: result 
+
+- name: Initialize lvm_vg name 
+  set_fact:
+    lvm_vg: "{{ result.name }}"
```

Comment:
You should probably add `when: lvm_vg == ""` here so it doesn't override/clobber a user-specified name.